app/controllers/posts_controller.rb

```diff
@@ -14,7 +14,7 @@ def new
   def show
     @user = User.find_by(id: params[:user_id])
     @post = @user.posts.find(params[:id])
-    @current_user = current_user
+    @currrent = current_user
     @like = Like.new
   end
```

Comment:
- [ ] It is required to use cancancan to add  post authorization to your post controller. This should be added on the first line of your controller. Something like this. :point_down: 

```
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
@Qoosim.
Thank you for reviewing my code. I really appreciate the time and effort you invested, but I guess the requirement is not to use CanCanCan to add post authorization but to set the delete post authorization when implementing the deleting post functionality (see image below):
![image](https://github.com/momo-87/blog-app/assets/122116375/85c38132-990d-4731-b287-e2d25d0e8447)

Below are the authorization I set in the Ability class:
![image](https://github.com/momo-87/blog-app/assets/122116375/5758e3d4-095a-4882-b489-a6c54ab1cc8a)